rdkit/ML/BuildComposite.py

```diff
@@ -657,7 +657,7 @@ def RunOnData(details, data, progressCallback=None, saveIt=1, setDescNames=0):
     message('\nEntire data set:')
     resTup = ScreenComposite.ShowVoteResults(
       range(data.GetNPts()), data, composite, nPossibleVals[-1], details.threshold)
-    nGood, nBad, nSkip, avgGood, avgBad, avgSkip, voteTab = resTup
+    nGood, nBad, nSkip, avgGood, avgBad, avgSkip, voteTab = resTup  # @UnusedVariable
```

Comment:
I think it would be useful to change the code at some point to used namedtuples.